### DISCUSSION TITLE
Having difficulty finding documentation for editing the config file.

### DISCUSSION THREAD

**Discussion by parthkarnawat** [roflcoopter/viseron#389] [Q&A] [answered]
Where can i find the documentation for editing the config.yaml file.

https://viseron.netlify.app/docs/documentation/configuration

The above link seems to be out of  date in terms of documentation.
I am using the following config file

`ffmpeg:
  camera:
    camera_one:
      name: Front Door
      host: 10.30.6.31
      port: 554
      path: /stream1
      username: ****
      password: ****

nvr:
  camera_one:`
and it doesn't seem to work. 
Some help will be appreciated, thanks.

**Comment by roflcoopter** [comment, marked as answer]
The documentation should be correct, however it points to the latest v2 beta of Viseron.
The UI based config editor is only available in Viseron v2 so you have to pull the `roflcoopter/viseron:2.0.0b3` tag to have the latest version 

There are two ways to edit the config, either through the UI or by editing the config.yaml file directly.

What does your Docker run command look like?

**Reply by parthkarnawat** [reply]
I have followed the documentation for installation of viseron. I used the 64 bit linux machine with nvidida gpu example in the documentation.

> docker run --rm \
  -v {recordings path}:/recordings \
  -v {config path}:/config \
  -v /etc/localtime:/etc/localtime:ro \
  -p 8888:8888 \
  --name viseron \
  --runtime=nvidia \
  roflcoopter/amd64-cuda-viseron:latest

I did remove the --rm command and replaced the paths.

When i run docker  images command i get the following output.

> REPOSITORY                       TAG                       IMAGE ID       CREATED         SIZE
nvidia/cuda                      11.6.2-base-ubuntu20.04   bc3a8621a995   2 weeks ago     154MB
roflcoopter/amd64-cuda-viseron   latest                    e4caea5addae   11 months ago   6.45GB

**Reply by roflcoopter** [reply]
Okay, you need to use the `roflcoopter/amd64-cuda-viseron:2.0.0b3` tag to get the latest version which is reflected in the docs

**Reply by parthkarnawat** [reply]
Thank you! Now it works. It would be nice for new users if you could either update the docker hub or the documentation to use the `roflcoopter/amd64-cuda-viseron:2.0.0b3` tag.

**Reply by roflcoopter** [reply]
Yeah i understand the confusion. The documentation site is brand new for Viseron v2, but v2 is not fully released yet, its in its beta stage.
When v2 is released, the `latest` tag will be updated and will match with the docs.
This is why there is some confusion currently

The docs for v1 is in the README in the master branch.
